api/utils/error.js

```diff
@@ -0,0 +1,21 @@
+/* 
+ * simply wrapped asynchronous function inside of the catchAsync function.
+ * This function will return a new anonymous function, 
+ * which will then be assigned to asynchronous contoller
+ */
+const catchAsync = func => {
+	return (req, res, next) => {
+		func(req, res, next).catch((error) => next(error))
+	}
+}
+
+const globalErrorHandler = (err, req, res, next) => {
+	console.error(err.stack)
+
+	err.statusCode = err.statusCode || 500;
+
+	res.status(err.statusCode).json({ message: err.message })
+}
+
+
+module.exports = { catchAsync, globalErrorHandler }
```

Comment:
- 들여씌기 간격 확인해 주세요.
- 코드들 사이에 간격은 굳이 없어도 될 것 같습니다.
```suggestion
	console.error(err.stack)
	
	err.statusCode = err.statusCode || 500;
	res.status(err.statusCode).json({ message: err.message })
}

module.exports = { catchAsync, globalErrorHandler }
```